src/MemObject.h

```diff
@@ -192,11 +192,7 @@ class MemObject
     IRCB *ping_reply_callback;
     PeerSelector *ircb_data = nullptr;
 
-    struct abort_ {
-        abort_() { callback = nullptr; }
-        STABH *callback;
-        void *data = nullptr;
-    } abort;
+    AsyncCall::Pointer abortCallback;
```

Comment:
We should also document this arguably new data member. IIRC, these abort notifications are only used for StoreEntry writers. That is why just one data member is enough even though there could be many StoreEntry readers. If the code matches my recollection, I suggest the following comment:

```suggestion
    /// used for notifying StoreEntry writers about 3rd-party initiated aborts
    AsyncCall::Pointer abortCallback;
```